inst/include/temperature_component.hpp

```diff
@@ -145,6 +145,18 @@ class TemperatureComponent : public IModelComponent {
     unitval flux_interior; //!< heat flux into interior layer of ocean, W/m2
     unitval heatflux;      //!< heat flux into ocean, W/m2
 
+    // Potenital user defined model parameters
+    unitval lo_warming_ratio; //!< user defined land ocean warming ratio, unitless
```

Comment:
Let's provide some more information here? E.g. 
```cpp
// This ratio is the defined as ΔTL/ΔTO; for more information see
// https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/grl.50971
```

Reply:
☝️  and/or in the ini file?

Reply:
Can totally add that comment to the code. Would we want to switch from unitval U_UNITLESS to U_DTL_DTO or something? Or is it okay to leave at unitval unitless?

Reply:
I think unitless is correct.